static/styles/settings.css

```diff
@@ -386,6 +386,16 @@ td .button {
 .button,
 .input-group {
     margin: 0 0 20px;
+
+    &#user_deactivate_account_button {
+        margin-right: 12px;
+    }
+
+    &.deactivate_realm_button {
+        @media (width < $mm_min) {
```

Comment:
As I said before, it would nice to follow the convention where we keep media query CSS at the end.

Reply:
I have made a separate section for media querie.